### ISSUE TITLE
Allow CodeMirror to be disabled via configuration

### ISSUE THREAD

**Issue by natecavanaugh** [electricjs/electric#66]
Hey all,
Spoke with @Robert-Frampton and he said it should be doable, so just filing this here.

But on http://lexicondesign.io, we don't really need the CodeMirror functionality for highlighting code, so it would be nice to not have to have the script loaded or any of the related code executing.

Ideally, I see this as something like we do for Markdown, where in the config we maybe do something like this:
```js
codeMirror: false
```
and possibly
```js
codeMirror: function(config) {
return // An object or boolean
}
```

But those are just suggestions.

**Comment by protoEvangelion** [comment]
Hey @natecavanaugh ,

Once [this pr](https://github.com/electricjs/electric/pull/69) is merged you will be able to disable **codeMirror vendor packages** from being loaded by setting:

```js
codeMirror: false
```

in the `electric.config.js` file.

Let me know if you have any questions about it :)

**Comment by Robert-Frampton** [comment]
Released in v2.0.8.
